dvc/stage.py

```diff
@@ -229,7 +229,7 @@ def _changed_deps(self):
 
         if self.is_callback:
             logger.warning(
-                "DVC-file '{fname}' is a 'callback' stage "
+                "DVC-file '{fname}' is an 'callback' stage "
```

Comment:
why "an" though? Is "callback" a vowel sound?

Reply:
Woops, its because I had changed to "orphan" and then reverted to callback but missed the "an". Fixed in d0718cd9.